include/drivers/gnss/ublox_neo_m8_defines.h

```diff
@@ -0,0 +1,207 @@
+#define NBYTES_HIGH_ADDR			0xFD
```

Comment:
Would it be better if you have this file in enum instead of preprocessor macros?